### ISSUE TITLE
now dev on windows

### ISSUE THREAD

**Issue by lekoala** [juicyfx/now-php#50]
# Bug report

- Version: 0.0.9

## Description

I'm trying to use now dev on windows but i get these messages

```
> Ready! Available at http://localhost:3000
> Building now-php@0.0.9:api/index.php

        It looks like you don't have PHP on your machine.
        Learn more about how to run now dev on your machine.
        https://err.sh/juicyfx/now-php/now-dev-no-local-php
```

when visiting the url, i get vendor errors like this

`Warning: require_once(C:\Users\myusername\AppData\Local\Temp\zeit-fun-0a347cf3b4463\user\api/../vendor/autoload.php): failed to open stream: No such file or directory in C:\Users\myusername\AppData\Local\Temp\zeit-fun-0a347cf3b4463\user\api\index.php on line 7`

is windows supported ? nothing is mentionned in the url now-dev-no-local-php about windows. It seems to work more or less since I'm seeing a warning message, so maybe it's easy to fix.

**Comment by f3l1x** [comment]
Hi @lekoala. Installing composer dependencies is not supported on local development. Thing is, run php app via `php -S localhost:8000` is so easy.

**Comment by lekoala** [comment]
@f3l1x yes sure i guess it's not required as such, but it would be great if it worked

tried recently and i get this

TypeError: Cannot set property 'mode' of undefined
    at Object.getPhpFiles (C:\Users\MYUSER\AppData\Local\com.vercel.cli\Cache\dev\builders\node_modules\vercel-php\dist\utils.js:71:34)

it would be great somehow to have a way to set the executable. i have a php.exe working in my path but that doesn't seem to be enough :)
